packages/node-integration-tests/suites/public-api/OnUncaughtException/mimic-native-behaviour-additional-listener-test-script.js

```diff
@@ -0,0 +1,27 @@
+const Sentry = require('@sentry/node');
```

Comment:
super-l: we might want to rename this file as "mimic" is no longer part of the option name

Reply:
Actually, disregard; we still have some references to it. I'm fine with keeping the name (hence, resolving this)